From b41af23ee9b2d1268c4c24cfc22da270bc186f27 Mon Sep 17 00:00:00 2001
From: Tobias Wiese <tobias@tobiaswiese.com>
Date: Sun, 2 May 2021 21:07:38 +0200
Subject: [PATCH] Add lost spaces to parts that include literals. Closes
 #987930

At some locations e.g. in Bug title recommendations, literal parts
are mixed with other parts.
These parts had spaces escaped, which resulted in them not showing up in
the final document.

Signed-off-by: Tobias Wiese <tobias@tobiaswiese.com>
---
 source/pkgs.rst  | 21 ++++++++++-----------
 source/tools.rst |  2 +-
 2 files changed, 11 insertions(+), 12 deletions(-)

diff --git a/source/pkgs.rst b/source/pkgs.rst
index ed6f55e..c131649 100644
--- a/source/pkgs.rst
+++ b/source/pkgs.rst
@@ -25,7 +25,7 @@ including, but not limiting yourself to, the description of the package
 (so that others can review it), the license of the prospective package,
 and the current URL where it can be downloaded from.
 
-You should set the subject of the bug to ``ITP:``\ *foo*\ ``--``\ *short
+You should set the subject of the bug to ``ITP:`` *foo* ``--`` *short
 description*, substituting the name of the new package for *foo*. The
 severity of the bug report must be set to ``wishlist``. Please send a
 copy to ``debian-devel@lists.debian.org`` by using the X-Debbugs-CC
@@ -122,7 +122,7 @@ minimum, you should try the following activities (you'll need to have an
 older version of the same Debian package around):
 
 -  Run ``lintian`` over the package. You can run ``lintian`` as follows:
-   ``lintian -v``\ *package-version*\ ``.changes``. This will check the
+   ``lintian -v`` *package-version*\ ``.changes``. This will check the
    source package as well as the binary package. If you don't understand
    the output that ``lintian`` generates, try adding the ``-i`` switch,
    which will cause ``lintian`` to output a very verbose description of
@@ -387,7 +387,7 @@ uploading to ``ftp.upload.debian.org`` in upload-directory
 ``DELAYED/``\ *X*\ ``-day`` (*X* between 0 and 15). 0-day is uploaded
 multiple times per day to ``ftp.upload.debian.org``.
 
-With dput, you can use the ``--delayed``\ *DELAY* parameter to put the
+With dput, you can use the ``--delayed`` *DELAY* parameter to put the
 package into one of the queues.
 
 .. _s5.6.4:
@@ -722,7 +722,7 @@ applied to the tag ``fixed-in-experimental``.
 
 If you happen to mistype a bug number or forget a bug in the changelog
 entries, don't hesitate to undo any damage the error caused. To reopen
-wrongly closed bugs, send a ``reopen``\ *XXX* command to the bug
+wrongly closed bugs, send a ``reopen`` *XXX* command to the bug
 tracking system's control address, ``control@bugs.debian.org``. To close
 any remaining bugs that were fixed by your upload, email the
 ``.changes`` file to *XXX*\ ``-done@bugs.debian.org``, where *XXX* is
@@ -1068,7 +1068,7 @@ is an old compatibility library which is no longer required), you need
 to file a bug against ``ftp.debian.org`` asking that the package be
 removed; as with all bugs, this bug should normally have normal
 severity. The bug title should be in the form
-``RM:``\ *package[architecture list]*\ ``--``\ *reason*, where *package*
+``RM:`` *package [architecture list]* ``--`` *reason*, where *package*
 is the package to be removed and *reason* is a short summary of the
 reason for the removal request. *[architecture list]* is optional and
 only needed if the removal request only applies to some architectures,
@@ -1121,8 +1121,7 @@ and https://qa.debian.org/howto-remove.html\ .
 If in doubt concerning whether a package is disposable, email
 ``debian-devel@lists.debian.org`` asking for opinions. Also of interest
 is the ``apt-cache`` program from the ``apt`` package. When invoked as
-``apt-cache
-showpkg``\ *package*, the program will show details for *package*,
+``apt-cache showpkg`` *package*, the program will show details for *package*,
 including reverse depends. Other useful programs include
 ``apt-cache rdepends``, ``apt-rdepends``, ``build-rdeps`` (in the
 ``devscripts`` package) and ``grep-dctrl``. Removal of orphaned packages
@@ -1190,7 +1189,7 @@ If you can no longer maintain a package, you need to inform others, and
 see that the package is marked as orphaned. You should set the package
 maintainer to ``Debian QA Group <packages@qa.debian.org>`` and submit a
 bug report against the pseudo package ``wnpp``. The bug report should be
-titled ``O:``\ *package*\ ``--``\ *short description* indicating that
+titled ``O:`` *package* ``--`` *short description* indicating that
 the package is now orphaned. The severity of the bug should be set to
 ``normal``; if the package has a priority of standard or higher, it
 should be set to important. If you feel it's necessary, send a copy to
@@ -1200,7 +1199,7 @@ the message's subject won't indicate the bug number).
 
 If you just intend to give the package away, but you can keep
 maintainership for the moment, then you should instead submit a bug
-against ``wnpp`` and title it ``RFA:``\ *package*\ ``--``\ *short
+against ``wnpp`` and title it ``RFA:`` *package* ``--`` *short
 description*. ``RFA`` stands for ``Request For Adoption``.
 
 More information is on the `WNPP web
@@ -1361,7 +1360,7 @@ you should check or be aware of.
    is usually incorrect.
 
 3. Make sure your source package is correct. Do
-   ``dpkg-source -x``\ *package*\ ``.dsc`` to make sure your source
+   ``dpkg-source -x`` *package*\ ``.dsc`` to make sure your source
    package unpacks properly. Then, in there, try building your package
    from scratch with ``dpkg-buildpackage``.
 
@@ -1408,7 +1407,7 @@ need to touch any of the files in the source package. This includes
 ``debian/changelog``.
 
 The way to invoke ``dpkg-buildpackage`` is as ``dpkg-buildpackage -B
--m``\ *porter-email*. Of course, set *porter-email* to your email
+-m`` *porter-email*. Of course, set *porter-email* to your email
 address. This will do a binary-only build of only the
 architecture-dependent portions of the package, using the
 ``binary-arch`` target in ``debian/rules``.
diff --git a/source/tools.rst b/source/tools.rst
index 9bd40b1..aabdf6f 100644
--- a/source/tools.rst
+++ b/source/tools.rst
@@ -21,7 +21,7 @@ to the exclusion of a competing tool.
 Most of the descriptions of these packages come from the actual package
 descriptions themselves. Further information can be found in the package
 documentation itself. You can also see more info with the command
-``apt-cache show``\ *package-name*.
+``apt-cache show`` *package-name*.
 
 .. _tools-core:
 
-- 
2.29.2

